src/components/Cards/ProvidersCard.tsx

```diff
@@ -45,22 +35,22 @@ export function ProvidersCard() {
   };
 
   const miniSearch = new MiniSearch({
-    fields: ['name'],
-    storeFields: ['name'],
+    fields: ["name"],
```

Comment:
This is just my prettier again, lmk if you want me to change it back to single quotes. Same thing applies to when I add semicolons and other stylistic changes throughout the file. They were made programmatically and I am happy to flip them back.